src/types/sdk.ts

```diff
@@ -185,6 +190,10 @@ export type PathWithPosition = {
   // if debug flag set, these will be populated
   settlerExecutionParams?: SettlerExecutionParams[];
   swapAmountInMilliBps?: number;
+  protocolShareBps: number;
+  senderShareBps: number;
+  protocolShare: FeeAmount;
+  senderShare: FeeAmount;
```

Comment:
Instead of 4 entries here, what about something like:

```typescript
fees: {
    protocol: MigrationFees; // renaming FeeAmount type to MigrationFees
    sender: MigrationFees;
    total: MigrationFees;
}
```